packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -103,6 +103,10 @@ class _MulticastCanvas implements Canvas {
     _screenshot.clipRect(rect, clipOp: clipOp, doAntiAlias: doAntiAlias);
   }
 
+  Rect getCurrentClipBounds() {
```

Comment:
maybe add a trivial test that checks these are Rect.zero and Identity?

Reply:
And then amend the tests when they return actual values?

Reply:
if that would break the roll, then probably not. i'd say its tested by way of allowing the engine PR to land by unbreaking those tests. You could also implement noSuchMethod

Reply:
I didn't think of NSM...

Reply:
May want to have an `@override` on these with an ignore so that the analyzer will ignore it now but accept it when they're actually overriding something.

NSM is fine too

Reply:
I went the NSM route as it's simpler. We might want to consider leaving it there permanently so that this only becomes a problem when new testing code needs the new implementations (in these screenshot tests).